web/src/main/java/org/openmrs/web/WebDaemon.java

```diff
@@ -10,63 +10,52 @@
 package org.openmrs.web;
 
 import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
 
-import org.openmrs.api.context.Context;
 import org.openmrs.api.context.Daemon;
 import org.openmrs.module.ModuleException;
 import org.openmrs.util.DatabaseUpdateException;
 import org.openmrs.util.InputRequiredException;
 
+import java.util.concurrent.ExecutionException;
+
 /**
  * This class provides {@link Daemon} functionality in a web context.
  * 
  * @since 1.9
  */
-public class WebDaemon extends Daemon {
+public final class WebDaemon {
```

Comment:
Technically, this is a breaking change, since `WebDaemon` used to also export the public methods of `Daemon`. However, we never used that internally and I could not find any code on GitHub that made use of `WebDaemon` (other than various forks of this repo).